mesheryctl/internal/cli/root/system/start.go

```diff
@@ -98,155 +134,214 @@ func start() error {
 		return errors.Wrap(err, "error processing config")
 	}
 
-	// Viper instance used for docker compose
-	utils.ViperCompose.SetConfigFile(utils.DockerComposeFile)
-	err = utils.ViperCompose.ReadInConfig()
-	if err != nil {
-		return err
-	}
+	// TODO: Centralise docker-compose.yaml file pull for this and reset
+	// get the platform, channel and the version of the current context
+	currPlatform := mctlCfg.GetContextContent().Platform
+	RequestedAdapters := mctlCfg.GetContextContent().Adapters // Requested Adapters / Services
+	// currChannel := mctlCfg.Contexts[currentContext].Channel
+	// currVersion := mctlCfg.Contexts[currentContext].Version
+	// fileURL := ""
```

Comment:
Pick up the current platform and the requested adapters. Moved the requested adapters out of the `docker` case since it is common for both deploying to docker and kubernetes.